wallet/update.go

```diff
@@ -158,12 +158,18 @@ func applyChainState(tx UpdateTx, address types.Address, cau chain.ApplyUpdate)
 	utxoValues := make(map[types.SiacoinOutputID]types.Currency)
 
 	cau.ForEachSiacoinElement(func(se types.SiacoinElement, spent bool) {
-		if se.SiacoinOutput.Address != address || ephemeral[types.Hash256(se.ID)] {
+		if se.SiacoinOutput.Address != address {
 			return
 		}
 
 		// cache the value of the utxo to use when calculating outflow
 		utxoValues[types.SiacoinOutputID(se.ID)] = se.SiacoinOutput.Value
```

Comment:
It'd be better to move this up above the ephemeral check instead. We could pass the ephemeral outputs to the data store, but then we would need to pass that context to the store so it can handle them properly. If you're doing a balance calculation, that can get annoying.

Reply:
Disregard this. Missed that the check still exists